app/assess/data.py

```diff
@@ -178,10 +180,10 @@ def get_application(identifier: str) -> Union[Application, None]:
 
 def get_application_status(application_id: str) -> Union[Application, None]:
     application_status_endpoint = (
-        APPLICATION_STORE_API_HOST
+        Config.APPLICATION_STORE_API_HOST
         + APPLICATION_STATUS_ENDPOINT.format(application_id=application_id)
     )
-    print(application_status_endpoint)
+    current_app.logger.debug(application_status_endpoint)
```

Comment:
this might be a hangover - not sure it provides much context

Reply:
Yep for testing, should prob just be removed

Reply:
It's gone